pkg/skaffold/runner/deploy.go

```diff
@@ -41,7 +43,9 @@ func (r *SkaffoldRunner) Deploy(ctx context.Context, out io.Writer, artifacts []
 	}
 
 	if r.imagesAreLocal && len(artifacts) > 0 {
-		color.Green.Fprintln(out, "   local images can't be referenced by digest. They are tagged and referenced by a unique ID instead")
+		logrus.Debugln("Local images can't be referenced by digest.")
```

Comment:
This should be a single message.  But is it actually necessary?  It's not triggered by anything that the user has done.

Reply:
This is helpful to understand why the tags are not used locally.